networksurvey/src/main/java/com/craxiom/networksurvey/services/NetworkSurveyService.java

```diff
@@ -729,16 +733,19 @@ public Boolean toggleCellularLogging(boolean enable)
 
             Timber.i("Toggling cellular logging to %s", enable);
 
-            final boolean successful = cellularSurveyRecordLogger.enableLogging(enable);
+            final boolean successful = cellularSurveyRecordLogger.enableLogging(enable) &&
+                    phoneStateRecordLogger.enableLogging(enable);
```

Comment:
We probably need to handle the scenario where one logging turns on successfully, and the other one does not. In that event we need to turn off logging for both since we are only returning successful when logging starts for both.

Reply:
Updated to handle that scenario.